scripts/compile_solvers.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
+# Make a directory to work in
+export IDAES_SRC=`pwd`
```

Comment:
You are supposed to be in ```idaes-ext``` now, since I moved the ```src``` directory from ```idaes-dev``` to ```idaes-ext```.  I thought I updated the readme, but I'll check on it.

Reply:
Ah okay, I see it now. I think I was confused by the `IDAES_SRC` variable. Probably just me

Reply:
No, I see how that is confusing now that you mention it.  I'll change it to IDAES\_EXT, some things evolved in way that they don't make much sense anymore.